rust/src/common/config.rs

```diff
@@ -10,6 +9,7 @@ use log::debug;
 use rand::rng;
 use rand::seq::SliceRandom;
 
+use crate::HdfsError;
```

Comment:
Lint failing

Reply:
Do you think add Justfile or pre-commit's YAML file is a good idea?

- Justfile helps us know common commands.
- pre-commit helps us run some lint rules before we enter `git commit`.